### ISSUE TITLE
Mute background music when jingle is playing

### ISSUE THREAD

**Issue by PFLI2** [open-goal/jak-project#2253]
### What feature is your idea related to?

As a new player I can't help but notice that the celebration of picking up the game's main collectable is quite annoyingly undercut by the stage's music continuing on. I propose an option be added to completely mute the main music track during these celebrations.

### Describe the solution you'd like.

A toggle to completely mute all music when a jingle is playing (such as upon collecting a power cell) to avoid the tracks overlapping. It can then fade back in as the player regains control.

**Comment by ZippyZolton** [comment]
I'm fairly certain it does mute, or at least turn off most of the music tracks.

**Comment by Zedb0T** [comment]
It kind of dampers them, the only time it turns them off is if you are in a scene that already has its own unique music built in.

https://www.youtube.com/watch?v=205KdysMFtk

**Comment by ManDude** [comment]
Yeah there's a separate setting for music volume during movies and it is set to like 30% or something.
